--- ./drivers/char/random.c.orig	2019-12-21 10:57:45.000000000 +0100
+++ ./drivers/char/random.c	2019-12-26 10:05:26.135590558 +0100
@@ -428,7 +428,7 @@
  * its value (from 0->1->2).
  */
 static int crng_init = 0;
-#define crng_ready() (likely(crng_init > 1))
+#define crng_ready() (likely(crng_init >= 1))
 static int crng_init_cnt = 0;
 static unsigned long crng_global_init_time = 0;
 #define CRNG_INIT_CNT_THRESH (2*CHACHA20_KEY_SIZE)
